src/components/Loader/PersonLink/PersonLink.tsx

```diff
@@ -0,0 +1,23 @@
+import classNames from 'classnames';
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { Person } from '../../../types';
+
+type PersonLinkProps = {
+  people: Person,
```

Comment:
The prop should be named 'person' instead of 'people', since it expects a single person object.